src/lib/bundle.js

```diff
@@ -24,31 +24,22 @@ export default function bundle (bundleName, fetchOptions, _config) {
     Promise.all(
       config.languages.map(language => fetchData(config, language, fetchOptions))
     ).then((datas) => {
-      datas.forEach(saveFile)
+      // for each lang save files
+      datas.forEach(saveDataToFile)
```

Comment:
This runs asynchronously, which means the log following it is executed before it.
We can use a `map` to fire all asynchronous `saveDataToFile()`-calls and put them in an array. Then we use `Promise.all(arrayOfPromises)` to resolve them.

```js

Promise.all( // resolves an array of promises
  datas.map((data) => { // returns array of all promises from all saveDataToFile()-calls
    return saveDataToFile(data)
  })
).then(() => log.success('DONE.')) // is called when all promises are resolved (here: all files are saved)
```

Reply:
This applies for all other changes as well ✨